cypress/integration/mob-gallery-spec.js

```diff
@@ -0,0 +1,89 @@
+import { Preview } from '../page-objects/preview-page'
```

Comment:
change the name for an explicit  name, what is mob?

Reply:
mob refers to mobile. I tried to keep the name short.

Reply:
don't save on characters , always make names explicit

Reply:
Made the changes accordingly.